docs/markdown/axes.md

```diff
@@ -74,3 +74,18 @@ Height of the axis in pixels. **Already set by default**, but can be overriden b
 
 #### animation (optional)
 See the [XYPlot](xy-plot.md)'s `animation` section for more information.
+
+### style (optional)
```

Comment:
should use consistent number of headings, and wording (`Style` instead of `style` in `docs/markdown/hint.md`)